backend/tests/integration/records/serializers/json/test_jobs.py

```diff
@@ -185,35 +193,42 @@ def test_jobs_search_json(inspire_app, datadir):
     assert expected_updated == response_updated
 
 
-def test_jobs_search_json_can_edit(inspire_app):
-    headers = {"Accept": "application/json"}
+@mock.patch("flask_login.utils._get_user")
+def test_jobs_search_json_can_edit(mock_current_user, inspire_app):
+    headers = {"Accept": "application/vnd+inspire.record.ui+json"}
+    user_orcid = "0000-0002-9127-1687"
 
-    user = create_user(email="harun@cern.ch")
     create_record(
-        "job", data={"status": "open", "acquisition_source": {"email": "harun@cern.ch"}}
+        "job", data={"status": "open", "acquisition_source": {"orcid": user_orcid}}
     )
     create_record(
-        "job", data={"status": "open", "acquisition_source": {"email": "guy@cern.ch"}}
+        "job",
+        data={
+            "status": "open",
+            "acquisition_source": {"orcid": "0100-0002-9127-1687"},
+        },
     )
-
+    user_orcid = "0000-0002-9127-1687"
+    current_oauthclient.signup_handlers["orcid"] = {"view": True}
+    jobs_author = create_user(role="user", orcid=user_orcid, allow_push=True)
+    mock_current_user.return_value = jobs_author
     with inspire_app.test_client() as client:
-        login_user_via_session(client, email=user.email)
-        response = client.get("/jobs", headers=headers)
+        response = client.get(f"/jobs/", headers=headers)
 
     hits = response.json["hits"]["hits"]
 
     own_job_metadata = next(
         hit["metadata"]
         for hit in hits
-        if hit["metadata"]["acquisition_source"]["email"] == user.email
+        if hit["metadata"]["acquisition_source"]["orcid"] == user_orcid
     )
     another_job_metadata = next(
         hit["metadata"]
         for hit in hits
-        if hit["metadata"]["acquisition_source"]["email"] != user.email
+        if hit["metadata"]["acquisition_source"]["orcid"] != user_orcid
     )
 
-    assert "can_edit" not in another_job_metadata
+    assert not another_job_metadata["can_edit"]
```

Comment:
why you changed `assert "can_edit" not in another_job_metadata`? :)

Reply:
Cause it's visible (as in seminars; btw. variable names in the test are wrong).
https://github.com/inspirehep/inspirehep/blob/05c189b27610d8665c0aaae982a25734e1bce9dc/backend/tests/integration/records/serializers/json/test_seminars.py#L218-L230

In literature it's not visible but there is used two step serialisation
https://github.com/inspirehep/inspirehep/blob/05c189b27610d8665c0aaae982a25734e1bce9dc/backend/inspirehep/records/marshmallow/literature/ui.py#L191-L205

So should I make it as in literature (or hide it somehow if it's ```False```) or we keep it as in seminars?

Reply:
Thanks for explanation, let's keep it as seminars